spec/models/manageiq/providers/autosde/inventory_spec.rb

```diff
@@ -5,23 +5,34 @@
                       :hostname => Rails.application.secrets.autosde[:appliance_host])
   end
 
+  let(:first_wwpn) { "2100000E1EE89D90" }
+  let(:ems_ref) { "a785c49a-2f3c-4e6d-8ace-879dfa1719a2" }
```

Comment:
Don't want to drag this out any longer, can be fixed in a follow-up but this should be `:physical_storage_ems_ref` or something else that indicates what it is for